enclave_build/src/main.rs

```diff
@@ -133,9 +146,26 @@ fn main() {
     let signing_certificate = matches
         .value_of("signing_certificate")
         .map(|val| val.to_string());
-    let private_key = matches
-        .value_of("private_certificate")
+    let kms_key_arn = matches.value_of("kms-key-arn");
+    let kms_key_region = matches.value_of("kms-key-region");
+    let private_key_path = matches
+        .value_of("private_key")
         .map(|val| val.to_string());
+
+    let signing_key = match (kms_key_arn, private_key_path) {
```

Comment:
This shouldn't just be a key path. It should accept PKCS#11 URIs and conform to david.woodhou.se/draft-woodhouse-cert-best-practice.html